contracts/utils/BaseCompanion.sol

```diff
@@ -143,7 +143,7 @@ abstract contract BaseCompanion is SimulationAdapter, RevokableWithGovernor, Pay
    * @param _newSwapper The address of the new swapper
    * @param _newAllowanceTarget The address of the new allowance target
    */
-  function setSwapper(address _newSwapper, address _newAllowanceTarget) external onlyGovernor {
+  function setSwapper(address _newSwapper, address _newAllowanceTarget) external {
```

Comment:
By doing this, we are exposing ourselves to a big vulnerability. Anyone will be able to change the contract we use for swapping